db/migrate/20191203043958_create_reviews.rb

```diff
@@ -0,0 +1,11 @@
+class CreateReviews < ActiveRecord::Migration[5.1]
+  def change
+    create_table :reviews do |t|
+      t.string :title
+      t.integer :rating
+      t.string :content
+      t.string :image_path
+      t.references :shelter, foreign_key: true
```

Comment:
Let's add `t.timestamps` to this migration for consistency with the other ones.

Reply:
This has been changed in the actual file, not sure why it is not being represented here.

Reply:
Completed.